File: week6/pset6/sentimental-mario-less/mario.py
from cs50 import get_int

# Prompt user for pyramid height
height = 0

while height < 1 or height > 8:
    height = get_int("Height? ")

# Print pyramid
# Rows

for i in range(height):

    # print spaces
    spaces = height - (i+1)
    for j in range(spaces):
        print(" ", end="")

    # print '#'
    for k in range(i + 1):
        print("#", end="")

    print()

File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
// Prompt user for pyramid height
    int height = 0;
    do
    {
        height = get_int("Height? ");
    }
    while (height < 1 || height > 8);
//    printf("Stored: %i\n", height);

// Print pyramid
    int i = 1;
// Rows
    for (int n = height; n > 0; n--)
    {
        int j;
// spaces and #
        for (j = 0; j < n - 1; j++)
        {
            printf(" ");
        }
        int k = j;
        while (height - k > 0)
        {
            printf("#");
            k++;
        }
        printf("\n");
    }
}